src/@context/Web3Provider.tsx

```diff
@@ -0,0 +1,52 @@
+import { WagmiProvider, createConfig } from 'wagmi'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { ConnectKitProvider, getDefaultConfig } from 'connectkit'
```

Comment:
question(blocking): Can we ignore those warnings?

Module '"wagmi"' has no exported member 'WagmiProvider'.
Module '"wagmi"' has no exported member 'createConfig'.
Module '"connectkit"' has no exported member 'getDefaultConfig'.

Reply:
thanks. accidental commit of file. its removed again